Added user agent

3sier opened the pull request:
![image](https://github.com/Vsimpro/kiosk.vsim.xyz/assets/85907776/f09964d9-eb2e-44d9-9400-2c3e4351fdb2)

3sier committed:
Added user agent

Vsimpro commented:
Hey man! Love the idea. If possible could you add your name to the code, and check CONTRIBUTING.md to make sure your commit complies? Thanks in advance.

3sier committed:
Merge branch 'Vsimpro:main' into feature/user_agent

3sier commented:
Should i put it in another .html or in the index.html file?

Vsimpro commented:
For this the main page is okay for now I feel like.

3sier committed:
Update user agent section

Updated the user agent section in index.html to include a comment explaining how to get the user agent of the browser. The comment was written by @3sier.
